docs/howto/use-extensions/use-the-gnome-extension.rst

```diff
@@ -0,0 +1,26 @@
+.. _use-the-gnome-extension:
+
+Use the Gnome extension
+=======================
+
+To use the :ref:`gnome-extension` with an app, add it to the app's ``extensions``
+key in the snap recipe. For example:
+
+.. code:: yaml
+
+    apps:
+      tali:
+        extensions: [gnome]
+        command: usr/bin/tali
+
+For a comprehensive example of snap recipe that includes the extension, see
+:ref:`example-gtk4-app`.
+
+
+Additional interfaces
+---------------------
+
+When you include this extension, a number of :ref:`plugs
+<gnome-extension-included-plugs>` are automatically opened, so you won't need to declare these if needed.
```

Comment:
This might be a good place to link to [expand-extensions](https://canonical-snapcraft--5212.com.readthedocs.build/en/5212/reference/commands/expand-extensions/) and mention that users can see the plugs that the gnome extension configures using that command.

Reply:
Ideally we'd have a how-to for using expand-extensions that we could link to, but I don't believe we have one right now.

Reply:
Added, somewhat sloppily: f296e3f

Agreed that it needs to be addressed more structurally. Tracking that in CRAFT-4053.